.azure-pipelines/client.yml

```diff
@@ -34,6 +36,10 @@ jobs:
           twine check $(Build.ArtifactStagingDirectory)/*
         displayName: 'Verify Readme'
 
+      - script: 'echo "##vso[build.addbuildtag]$(ReleaseTag)"'
```

Comment:
Why does what with this special build variable? Are there any documents you can point me to?

Reply:
This `tags` the build. AKA  a `DevOps` tag. Found [here](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/authoring/commands.md#build-logging-commands)